<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>

<head>

  <meta content="text/html; charset=ISO-8859-1" http-equiv="content-type">
  <title>AGG Pretzel</title>


</head>

<body>
  <table style="text-align: left; width: 1024px; margin-left: auto; margin-right: auto;" border="0" cellpadding="2"
    cellspacing="2">

    <tbody>
      <tr>
        <td style="vertical-align: top;">
          <h2><img style="width: 94px; height: 86px; padding: 10px;" alt="pretzel-logo"
              src="https://plantinformatics.io/landingPageContent/pretzel-logo.png" align="left">AGG Pretzel<br>
          </h2>
          Pretzel is an interactive web application designed to enable rapid translation of genomics outputs for any species into concrete answers to research and breeding questions
          <br>
          <br>
          <br>
          <h3>Recent Updates</h3>
          <img src="https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/agg-landing-page.gif" alt="" style="width: 405px; height: 439px;" align="right">
          <ul>
            <li>25/05/2025 - New Pretzel version relaesed v3.8.0! New list of features <a href="https://github.com/plantinformatics/pretzel/releases/tag/v3.8.0">here</a></li>
            <li>30/04/2025 - New Pretzel functionality now available in release v3.7.3! Users can now search genotype data for accessions carrying user-defined haplotypes. See the new Use Case <a href="https://docs.plantinformatics.io/Use-Cases/Finding-genotyped-accessions-carrying-a-specific-haplotype/" target="_blank" rel="noopener noreferrer">Finding genotyped accessions carrying a specific haplotype</a> for more details
            <li>16/01/2025 - Added genotypes for 11,071 Chickpea AGG Plant genetic resources (<a href="https://doi.org/10.7910/DVN/SQFKJW" target="_blank" rel="noopener noreferrer">https://doi.org/10.7910/DVN/SQFKJW</a>) genotyped as part of the AGG Strategic Partnership, anchored against the CDC Frontier gnm3 assembly</li>
            <li>01/11/2024 - Added the 10+ Wheat Genomes from <i>Walkowiak et al. 2020</i>, including de novo gene annotations (<i><a href="https://doi.org/10.1101/2024.01.09.574802" target="_blank" rel="noopener noreferrer">White et al. 2024</a></i>), Wheat Barley 40k v1.1 mappings and an example of how it can all be used in </i><a href="https://docs.plantinformatics.io/User-Stories/User-story-3/" target="_blank" rel="noopener noreferrer">User Story 3 - From SSRs to pan genomes with Pretzel: Two decades of wheat yield QTLs on chromosome 7A</a></li>
            <li>23/10/2024 - Pretzel v3.0.0 released! This includes the VCF Search use case (see the documentation on how to use it <a href="https://docs.plantinformatics.io/Use-Cases/VCF-genotype-search/" target="_blank" rel="noopener noreferrer">here</a>) and a range of improvements and enhancements
            <li>17/09/2024 - Added <a href="https://docs.plantinformatics.io/User-Stories/User-story-2/" target="_blank" rel="noopener noreferrer">User Story 2 - Filtering AGG wheat accessions for stripe rust resistance gene Yr34/Yr48</a></li>
            <li>23/08/2024 - Added genotypes for 12,606 Hexaploid Wheat AGG Plant genetic resources (<a href="https://doi.org/10.7910/DVN/CRSI0B" target="_blank" rel="noopener noreferrer">https://doi.org/10.7910/DVN/CRSI0B</a>) genotyped as part of the AGG Strategic Partnership, anchored against the IWGSC RefSeq v2.1 assembly</li>
            <li>15/08/2024 - Added genotypes for 13,989 Barley AGG Plant genetic resources (<a href="https://doi.org/10.7910/DVN/H6SNVM">https://doi.org/10.7910/DVN/H6SNVM</a>) genotyped as part of the AGG Strategic Partnership, anchored against the Morex V3 assembly.</li> 
            <li>Create and account and try this example <a href="https://docs.plantinformatics.io/Use-Cases/Visualise-genotype-data-for-a-subset-of-accessions-around-specific-genomic-regions/">here</a></li>
          </ul>
          
          Current version is loaded with 
          <br>
          <ul>
              <li><i>Triticum aestivum - IWGSC_RefSeq_v2.1 - Genome</i></li>
              <li><i>Triticum aestivum - IWGSC_RefSeq_v2.1 - Markers - Wheat Barley 40k v1.1</i></li>
              <li><i>Triticum aestivum - IWGSC_RefSeq_v2.1 - Genes HC</i></li>
              <li><i>Triticum aestivum - IWGSC_RefSeq_v2.1 - Genes LC</i></li>
              <li><i>Triticum aestivum - IWGSC_RefSeq_v2.1 - Genotypes - AGG Release 1</i></li>
              <li><i>Triticum aestivum - IWGSC_RefSeq_v1.0 - Genome</i></li>
              <li><i>Triticum aestivum - IWGSC_RefSeq_v1.0 - Markers - Wheat Barley 40k v1.1</i></li>
              <li><i>Triticum aestivum - IWGSC_RefSeq_v1.0 - Genes HC</i></li>
              <li><i>Triticum aestivum - IWGSC_RefSeq_v1.0 - Genes LC</i></li>
              <li><i>Triticum aestivum - </i></li>10+ Wheat Genomes from <i>Walkowiak et al. 2020</i>, including de novo gene annotations (<i><a href="https://doi.org/10.1101/2024.01.09.574802" target="_blank" rel="noopener noreferrer">White et al. 2024</a></i>), Wheat Barley 40k v1.1 mappings</li>
              <li><i>Hordeum vulgare - MorexV3 - Genome</i></li> 
              <li><i>Hordeum vulgare - MorexV3 - Genes LC</i></li>
              <li><i>Hordeum vulgare - MorexV3 - Genes HC</i></li>
              <li><i>Hordeum vulgare - MorexV3 - Markers - Wheat Barley 40k v1.1</i></li>
              <li><i>Hordeum vulgare - MorexV3 - Genotypes - AGG Release 1</i></li>
              <li><i>Cicer arietinum - CDC Frontier gnm3 - Genome</i></li>
              <li><i>Cicer arietinum - CDC Frontier gnm3 - Genes</i></li>
              <li><i>Cicer arietinum - CDC Frontier gnm3 - Markers - Pulse 30K v1.0</i></li>
              <li><i>Cicer arietinum - CDC Frontier gnm3 - Genotypes - AGG Release 1</i></li>
              <li><i>Pisum sativum v1a - Field pea datasets described in <a href="https://docs.plantinformatics.io/User-Stories/User-story-1/">User Story 1</a></i></li>
          </ul>
          With the following coming soon:
          <br>
          <ul>
              <li>AGG pulses data</li>
          </ul>
          <br>
          <h5> 
            Look at a worked example of how Pretzel was used to <a href="https://docs.plantinformatics.io/User-Stories/User-story-1/">identify virus resistant PGRs in field peas.</a>
          </h5>
          <br>
          <h5>Documentation is available at 
            <a href="https://docs.plantinformatics.io">https://docs.plantinformatics.io</a>.
          </h5>
          <br>
          <h5>Please contact <a href="mailto:info@plantinformatics.io" target="_blank">info@plantinformatics.io</a> with
            any questions or queries.
          </h5>
          <br>
          <table style="text-align: left; margin-left: auto; margin-right: auto;" border="0"
            cellpadding="2" cellspacing="2">
            <tbody>
              <tr>
                <td>
                  <span>Pretzel is currently (2022-) funded as part of the Australian Grains Genebank Strategic Partnership, a $30M joint investment between the Victorian State Government and Grains Research and Development Corporation (GRDC) 
                    that aims to unlock the genetic potential of plant genetic resources for the benefit of the Australian grain growers.</span>
                    <br>
                </td>
              </tr>
              <tr>
                <td style="vertical-align: top; text-align: center;">
                  <a href="https://agriculture.vic.gov.au/crops-and-horticulture/the-australian-grains-genebank"></a><img alt="agvic" src="https://pretzel-images-public.s3.ap-southeast-2.amazonaws.com/agvic_grdc.svg" width="800">
                </td>
              </tr>
            </tbody>
          </table>
          <br>
          <br>
        </td>
      </tr>
    </tbody>
  </table>

  <br>


</body>

</html>
